### PR TITLE
Changes to userModel

### PR DESCRIPTION
**Pull request by swaroopdhadkar**

### CODE REVIEW

**Review by Trex-Hub — APPROVED** [review]
LGTM, Can Merge.

File: src/models/userModel.js
Comment (Trex-Hub) [inline src/models/userModel.js:1]: phone number can have fixed length here itself. If not you'll have to manage it on frontend

**Comment by swaroopdhadkar** [comment]
> LGTM, Can Merge.

Thanks Devendra.

File: src/models/userModel.js
Comment (swaroopdhadkar) [reply src/models/userModel.js:1]: Sure, will add a fixed length parameter here itself.

**Pull request merged**